tests/integration_test.go

```diff
@@ -27,160 +27,113 @@ const (
 	generateBlocksEnv = "GENERATEBLOCKS"
 	falseStr          = "false"
 
-	defaultXMRTakerDaemonEndpoint   = "http://localhost:5001"
-	defaultXMRTakerDaemonWSEndpoint = "ws://localhost:5001/ws"
-	defaultXMRMakerDaemonEndpoint   = "http://localhost:5002"
-	defaultXMRMakerDaemonWSEndpoint = "ws://localhost:5002/ws"
-	defaultCharlieDaemonWSEndpoint  = "ws://localhost:5003/ws"
+	defaultXMRTakerSwapdEndpoint   = "http://localhost:5001"
+	defaultXMRTakerSwapdWSEndpoint = "ws://localhost:5001/ws"
+	defaultXMRMakerSwapdEndpoint   = "http://localhost:5002"
+	defaultXMRMakerSwapdWSEndpoint = "ws://localhost:5002/ws"
+	defaultCharlieSwapdWSEndpoint  = "ws://localhost:5003/ws"
+
+	defaultXMRMakerMoneroEndpoint = "http://127.0.0.1:18083/json_rpc"
```

Comment:
The only change in the first block was replacing `Daemon` with `Swapd` to clarify which daemon was being referenced. 

The `defaultXMRMakerMoneroEndpoint` constant was moved from the `common` package. Normally, the end user would not specify a port that `monero-wallet-rpc` will provide RPC services on, but we provide the ability to set the port number. It was needed here in the integration tests, so that we could get Bob's (XMR Maker's) wallet address and mine some monero for him.